src/php/Filesystem/Infrastructure/RealFilesystem.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\Filesystem\Infrastructure;
+
+use Phel\Filesystem\Domain\FilesystemInterface;
+
+final class RealFilesystem implements FilesystemInterface
```

Comment:
What about renaming this class (and probably the interface?) to `RealFilesystemCollection`?

Reply:
To be honest, not sure if I really like the idea, but as a suggestion to take it into account. 😄

Reply:
I don't think I like the Collection suffix here, but thanks for the idea :)